Firestore/core/src/firebase/firestore/local/local_documents_view.h

```diff
@@ -103,6 +103,15 @@ class LocalDocumentsView {
   /** Queries the remote documents and overlays mutations. */
   model::DocumentMap GetDocumentsMatchingCollectionQuery(FSTQuery* query);
 
+  /**
+   * Find all `PatchMutation`s, and if their base documents are not in
+   * `existingDocs` add them to the returning `DocumentMap`. The returned map
+   * also has every document in `existingDocs`.
+   */
+  model::DocumentMap AddMissingBaseDocument(
```

Comment:
Nit: pluralize the name (`AddMissingBaseDocuments`).

Reply:
Done.

Reply:
Done.